client/we_meet_client/src/App.js

```diff
@@ -1,13 +1,20 @@
 import React, { Component } from 'react';
-import Events from './components/Events/Events'
+import './css/tailwind.css';
+import Events from './components/Events/Events';
+import AppHeader from './components/ui-components/AppHeader'
+import { Container } from 'tailwind-react-ui';
+
 
 class App extends Component {
   render() {
     return (
-      <div>
-        <h1>WeMeet</h1>
-        <Events />
-      </div>
+      <>
```

Comment:
I am seeing this empty tag for first time. Is that possible?

Reply:
This is a shorthand for `React.Fragment`

Reply:
Cool